target/linux/ramips/image/rt305x.mk

```diff
@@ -389,6 +392,25 @@ define Device/dlink_dir-320-b1
 endef
 TARGET_DEVICES += dlink_dir-320-b1
 
+define Device/dlink_dir-506l
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-506L
+  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-spi-dev \
+	kmod-usb-serial-option kmod-usb-net-cdc-ether comgt-ncm \
```

Comment:
I'm not familiar with it, do you really _require_ comgt-ncm here?
If yes, you may drop kmod-usb-serial-option here as well.

Reply:
I think we usually don't include any 3G/4G modem related packages unless the modem is actually built-in.
Just checked DIR-510L and DWR-116, and there are no WWAN-related packages included for those devices either, although they are advertised to support 4G (just because the OEM firmware has the drivers included).

Anyways, great to see a new PR for this device (I actually thought the first one was closed due to 32 MiB RAM back then).

Reply:
There is exactly _one_ device in entire ramips that adds comgt-ncm by default. If the modem is not built-in as stated, we should remove it here as well.